### PR TITLE
chore: add the kratos and oathkeeper config to the default values

### PR DESCRIPTION
**Pull request by openoms**
The Oathkeeper changes are based on:
https://github.com/GaloyMoney/galoy/pull/2103/files#diff-f2d3bed1938639bd048c1629be6fe575621bc4eb0cef967ab52d10f6ca96369e

Kratos:
https://github.com/GaloyMoney/galoy/blob/6f3bcdbb4bc24beb3546a5ade90a5384cc707072/dev/ory/kratos.yml

except:
`schemas`
`dsn`

A new postgresql instance is added to stand up before kratos and the `dsn` is configured accordingly.

### CODE REVIEW

**Review by krtk6160 — APPROVED** [review]

File: charts/galoy/Chart.yaml [hunk 1]
```diff
@@ -31,6 +31,11 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     version: 11.9.13
     condition: postgresql.enabled
+  - alias: kratospg
```
Comment (krtk6160) [inline charts/galoy/Chart.yaml:34]: should be kratos-pg imo

Comment (openoms) [reply charts/galoy/Chart.yaml:34]: yes, it was a problem in the templates/galoy-secrets.yaml

anyway the the kratos chart has an automigration init container which runs before the postgresql deployment in the same chatr so need to move the postgres instance out

File: charts/galoy/values.yaml [hunk 2]
```diff
@@ -708,23 +715,170 @@ kratos:
...
+                    config:
+                      # url: http://e2e-tests:4002/auth/after_settings_hooks
+                      url: http://invalid-because-we-dont-want-profile-to-be-updated
+                      method: POST
+                      body: file:///home/ory/body.jsonnet
+                      auth:
+                        type: api_key
+                        config:
+                          name: Authorization
+                          value: The-Value-of-My-Key
```
Comment (ntheile) [inline charts/galoy/values.yaml:801]: hey @openoms , i have been troubleshooting in my local dev charts environment and I think we need to read this value from the `callback_api_key` from here https://github.com/GaloyMoney/charts/blob/main/dev/galoy/main.tf#L314. I am not quite sure how to configure it here in the yaml file.

File: charts/galoy/values.yaml [hunk 3]
```diff
@@ -708,23 +715,170 @@ kratos:
...
+                    config:
+                      url: http://e2e-tests:4012/kratos/registration
+                      method: POST
+                      can_interrupt: true
+                      body: file:///home/ory/body.jsonnet # TODO: use a base64 encoding instead
+                      auth:
+                        type: api_key
+                        config:
+                          name: Authorization
+                          value: The-Value-of-My-Key
```
Comment (ntheile) [inline charts/galoy/values.yaml:842]: @openoms same here

**Pull request merged**